package.json

```diff
@@ -50,7 +50,8 @@
           "usergroups.collection.get",
           "module.checkin.enabled",
           "inventory.items.collection.get",
-          "inventory-storage.service-points.collection.get"
+          "inventory-storage.service-points.collection.get",
+          "accounts.collection.get"
```

Comment:
The description refers to `accounts.collection.get` and `lost-item-fees-policies.collection.get` (which I've observed an error for), yet this change only seems to include the former, is that intentional?

Reply:
Whoops; sorry; thanks; fixed.